### PR TITLE
fix: Introduce hard limit on event size after normalization

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

**Comment by untitaker** [comment]
This is kind of a shitty solution but I would prefer to postpone perf improvements once we delete the old code. I assume this would involve a lot of changes to the `EventManager` API.

File: src/sentry/web/api.py [hunk 1]
```diff
@@ -549,6 +549,12 @@ def process(self, request, project, key, auth, helper, data, attachments=None, *
         self.pre_normalize(event_manager, helper)
         event_manager.normalize()
 
+        data = event_manager.get_data()
+        dict_data = dict(data)
+
+        if len(json.dumps(dict_data)) > 10000000:
+            raise APIForbidden("Event size got 10MB after normalization.")
```
Comment (mitsuhiko) [inline src/sentry/web/api.py]: Make it "exceeded 10MB".

**Review by mitsuhiko — DISMISSED** [review]

File: src/sentry/web/api.py (hunk 1 above)
Comment (untitaker) [reply src/sentry/web/api.py]: done

**Comment by mitsuhiko** [comment]
Going to wait for the resolution fo the DACI with this.

**Review by mitsuhiko — APPROVED** [review]
lgtm me now, but let's not push this out until next week.

**Pull request merged**